### PR TITLE
Documentation: Config

### PR DESCRIPTION
**Pull request by mejroslav**
Add docstrings to `asab.Config`, `asab.ConfigObject` and various utility functions.

image

### CODE REVIEW

File: asab/config.py [hunk 1]
```diff
@@ -206,6 +217,26 @@ def _load(self):
 
 
 	def _include_from_zookeeper(self, zkurl):
+		"""
```
Comment (mejroslav) [inline asab/config.py:220]: @mithunbharadwaj Could you please check this one? :)

Comment (eliska-n) [inline asab/config.py:220]: it can read the zkurl from environmental variable as well

Comment (mithunbharadwaj) [reply asab/config.py:220]: Looks okay to me.

Comment (mithunbharadwaj) [reply asab/config.py:220]: @mejroslav you can add this to doctoring too.

**Pull request merged**